src/bp-activity/bp-activity-functions.php

```diff
@@ -1011,6 +1123,226 @@ function bp_activity_get_actions_for_context( $context = '' ) {
 	return $actions;
 }
 
+/** Reactions ****************************************************************/
+
+/**
+ * React to an activity.
+ *
+ * @since 14.0.0
+ *
+ * @param array|string $args {
+ *     An array of arguments.
+ *     @type int        $user_id           Optional. The ID of the user reacting.
+ *                                         Defaults to the ID of the logged-in user.
+ *     @type object|int $activity          Required. The parent activity object or its ID.
```

Comment:
Better to say what object.
```suggestion
 *     @type BP_Activity_Activity |int $activity          Required. The parent activity object or its ID.
```

Reply:
Problem is the function to get the activity to add a like for into the corresponding `src/bp-activity/actions/like.php` file is `bp_activity_get_specific()` which is not returning a `BP_Activity_Activity` object but a standard one. So I'll leave it this way for now.